<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="EnterNotice">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="YourNoticeLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Your notice</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonPublish">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>300</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Publish</string>
    </property>
   </widget>
   <widget class="QLabel" name="NameOfMoteLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name of note</string>
    </property>
   </widget>
   <widget class="QLabel" name="KeyLabel">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QLabel" name="NotesLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ListArchive">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>100</y>
      <width>256</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ArchieLabel">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>80</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Archive</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ShownNote">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>321</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="TextOfNoteLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Text of note</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ListNotes">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>256</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonCommit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>520</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Commit changes</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ActionText">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>560</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="ActionLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>540</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Action</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EnterFilter">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>50</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="FilterLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter key</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EnterName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="EnterKey">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1103</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
